source/draw/Canvas.ooc

```diff
@@ -17,7 +17,7 @@
 use base
 use geometry
 use collections
-import Image, Pen
+import Image, Pen, DrawState
```

Comment:
Probably not your fault originally, but please put these on three separate lines.